config/jobs/kubernetes-sigs/structured-merge-diff/structured-merge-diff-ci.yaml

```diff
@@ -0,0 +1,17 @@
+periodics:
+- name: ci-structured-merge-diff-test
+  interval: 1h
+  labels:
+    preset-service-account: "true"
+  spec:
```

Comment:
For this job please add `decorate: true` above the `spec` field, and this to specify the checkout:
```
extra_refs:
- org: kubernetes-sigs
  repo: structured-merge-diff
  base_ref: master
  path_alias: sigs.k8s.io/structured-merge-diff
```
Then make the args something like:
```
command:
- go
args:
- test
- ./...
```